selectrum.el

```diff
@@ -1084,6 +1084,14 @@ TABLE defaults to `minibuffer-completion-table'. PRED defaults to
         (let* ((prefix (get-text-property
                         0 'selectrum-candidate-display-prefix
                         candidate))
+               (isuffix (get-text-property
+                         ;; Internal property to display an additional
+                         ;; suffix before the actual suffix added via
+                         ;; public API. Currently only used for
+                         ;; displaying slashes of directories in file
+                         ;; completions.
+                         0 'selectrum--internal-candidate-display-suffix
+                         candidate))
```

Comment:
Hmmm. This seems slightly unfortunate. It feels kind of similar to some of the things going on in Ivy/Counsel, where the public API isn't expressive enough to implement all the needed features, and so some of the Ivy functions have special cases to handle specific callers from Counsel. In an ideal world, we'd be able to support multiple suffixes natively in some way through the public API.

I don't know if that's actually achievable---maybe this is one of those cases where you just have to hack it, because the alternative is too inconvenient---but I thought I would call attention to it since I think we want to try to avoid this kind of thing suffusing the entire codebase, since then we'll end up in an Ivy/Counsel sort of situation.

Reply:
Oops, hehe, I see you got rid of it in https://github.com/raxod502/selectrum/pull/262. That's what I get for being so behind on email!

Reply:
I'm happy it could be removed because of the reasons you mentioned above, there where some refactoring issues after I switched but I hope these have all been sorted out now.